ledger/metrics/collection/summary.js

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const http = require("http");
+
+const window = "6hours"
+
+const url = (target) => `http://localhost:2000/render?target=summarize(${target},%20%27${window}%27,%20%27last%27)&from=-${window}&format=json`
+
+function fetch(url) {
+    return new Promise((resolve, reject) => {
+        http.get(url, res => {
+            res.setEncoding("utf8");
+            let body = "";
+            res.on("data", data => {
+              body += data;
+            });
+            res.on("end", () => {
+              body = JSON.parse(body);
+              resolve(body);
+            });
+            res.on("error", (e) => {
+                reject(e);
+            });
+          });
+    })
+}
+
+function toDict(rawData) {
+    const result = {}
```

Comment:
```suggestion
    const result = {};
```

You may want to run `prettier` over all of this:

```
nix-shell -p nodePackages.prettier --run 'prettier --write ledger/metrics/collection/summary.js'
```